sdk/contentunderstanding/azure-ai-contentunderstanding/generated_samples/content_analyzers_create_or_replace.py

```diff
@@ -0,0 +1,106 @@
+from azure.ai.contentunderstanding.models import (
+    ContentAnalyzer,
+    ContentAnalyzerConfig,
+    FieldSchema,
+    FieldDefinition,
+    FieldType,
+    GenerationMethod,
+)
+from sample_helper import get_credential, save_response_to_file
+
+load_dotenv()
+
+
+# ---------------------------------------------------------------------------
+# Sample: Create custom analyzer using begin_create_or_replace API
+# ---------------------------------------------------------------------------
+# This sample demonstrates:
+# 1. Authenticate with Azure AI Content Understanding
+# 2. Create a custom analyzer with field schema using object model
+# 3. Wait for analyzer creation to complete
+# 4. Save the analyzer definition to a JSON file
+
+async def main() -> None:
+    endpoint = os.environ["AZURE_CONTENT_UNDERSTANDING_ENDPOINT"]
+    credential = get_credential()
+
+    async with ContentUnderstandingClient(endpoint=endpoint, credential=credential) as client, credential:
+        analyzer_id = f"sdk-sample-custom-analyzer-{int(asyncio.get_event_loop().time())}"
+        
+        # Create a custom analyzer using object model
+        custom_analyzer = ContentAnalyzer(
+            base_analyzer_id="prebuilt-documentAnalyzer",
+            description="Custom analyzer for extracting company information",
+            config=ContentAnalyzerConfig(
+                enable_formula=True,
+                enable_layout=True,
+                enable_ocr=True,
+                estimate_field_source_and_confidence=True,
+                return_details=True,
+            ),
+            field_schema=FieldSchema(
+                name="company_schema",
+                description="Schema for extracting company information",
+                fields={
+                    "company_name": FieldDefinition(
+                        type=FieldType.STRING,
+                        method=GenerationMethod.EXTRACT,
+                        description="Name of the company",
+                    ),
+                    "total_amount": FieldDefinition(
+                        type=FieldType.NUMBER,
+                        method=GenerationMethod.EXTRACT,
+                        description="Total amount on the document",
+                    ),
+                },
+            ),
+        )
+
+        print(f"🔧 Creating custom analyzer '{analyzer_id}'...")
+        poller = await client.content_analyzers.begin_create_or_replace(
+            analyzer_id=analyzer_id,
+            resource=custom_analyzer,
+        )
+        result = await poller.result()
+        print(f"✅ Analyzer '{analyzer_id}' created successfully!")
+        
+        # Clean up the created analyzer (demo cleanup)
+        print(f"🗑️  Deleting analyzer '{analyzer_id}' (demo cleanup)...")
+        await client.content_analyzers.delete(analyzer_id=analyzer_id)
+        print(f"✅ Analyzer '{analyzer_id}' deleted successfully!")
```

Comment:
I wonder if we too many print() statements.  They are obscuring the actual code.  I also find the emojis distracting, but that is subjective.